### DISCUSSION TITLE
I have had an tenaciously persistent issue with the colors of the History search feature. I wonder how you have set your colors for FZF?

### DISCUSSION THREAD

**Discussion by martisj** [PatrickF1/fzf.fish#317] [Q&A]
You see I have tried copying from: https://github.com/tinted-theming/base16-fzf/blob/main/fish/base16-gruvbox-dark-medium.fish and most of the colors worked fine. *But* the history preview window colors were *ALWAYS* (and I mean always) off. 
![image](https://github.com/PatrickF1/fzf.fish/assets/661994/00b0bdb8-0183-42fe-a6bb-fdf1d71f1f58)
The image where it says `locate pacman.conf` is clearly not the same as the rest of the prompt. Since this is similar to what is printed in the preview I reckon it should work to illustrate what I am struggling with. (see https://github.com/PatrickF1/fzf.fish/blob/46c7bc6354494be5d869d56a24a46823a9fdded0/functions/_fzf_search_history.fish#L26)
I should also mention that I have fiddled quite a bit with my colors... :sweat_smile: And it looks a lot like the colors from my Tokyo Night era. or maybe catppuccin.

**Comment by PatrickF1** [comment]
[Could it be the second part of this troubleshooting guide](https://github.com/PatrickF1/fzf.fish/wiki/Troubleshooting#search-history-doesnt-work)?
